openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -1124,21 +1123,43 @@ def analyse(where) -> dict:
         f3.savefig(savedir / "ligand_RMSD.png")
         plt.close(f3)
 
-        return {'structural_analysis': data}
+        # Save to numpy compressed format (~ 6x more space efficient than JSON)
+        np.savez_compressed(
+            shared / "structural_analysis.npz",
+            protein_RMSD=np.asarray(
+                data["protein_RMSD"], dtype=np.float32
+            ),
+            ligand_RMSD=np.asarray(
+                data["ligand_RMSD"], dtype=np.float32
+            ),
+            ligand_COM_drift=np.asarray(
+                data["ligand_wander"], dtype=np.float32
+            ),
+            protein_2D_RMSD=np.asarray(
+                data["protein_2D_RMSD"], dtype=np.float32
+            ),
+            time_ps=np.asarray(
+                data["time(ps)"], dtype=np.int32
+            ),
```

Comment:
Possible to walk through keys of `data` to avoid hardcodes of these fields?

Reply:
"ligand_wander" and "time(ps)" are the main issues here. We should revisit this in a future PR where we clean up structural analysis further.